changelog/17005.txt

```diff
@@ -0,0 +1,18 @@
+```release-note:change
+auth: `enable` command now returns an error for deprecated auth methods.
+```
+```release-note:change
+auth: `POST /sys/auth/:type` endpoint now reports an error for deprecated auth methods.
+```
+```release-note:change
+secrets: `enable` command now returns an error for deprecated secrets engines.
+```
+```release-note:change
+secrets: `POST /sys/mount/:type` endpoint now reports an error for deprecated auth secrets engines.
+```
+```release-note:improvement
+core: Add `VAULT_ALLOW_PENDING_REMOVAL_MOUNTS` to override Pending Removal shutdown and error on mount.
+```
+```release-note:improvement
+core: Handle and log deprecated builtin mounts
+```
```

Comment:
I am wondering if it is possible to reduce the changelog entries as some are related.

Would it be possible to have a single `auth` entry that mentions the change to the `enable` command noting that the underying `POST` to `/sys/auth/:type` is also changed? Similar question for the `secrets` entry?

Would it be possible to group the two `core` entries by adding a note about the `VAULT_ALLOW_PENDING_REMOVAL_MOUNTS` env var? My thought is that since this is net new behavior it might be throw off users if they are note aware of what **Pending Removal** means at quick glance.

Reply:
I was actually wondering if there was a way to compress these. I like the proposed solution, thanks!